images/jenkins/config/template.apko.yaml

```diff
@@ -1,35 +1,45 @@
 contents:
   packages:
     # jenkins and openjdk provided by extra_packages
+    - jenkins-docker
 
 accounts:
   groups:
     - groupname: jenkins
-      gid: 65532
+      gid: 1000
   users:
     - username: jenkins
-      uid: 65532
-  run-as: 65532
-
-work-dir: /app
+      gid: 1000
+      uid: 1000
+      homedir: "/var/jenkins_home"
+  # Match upstream
+  run-as: 1000
```

Comment:
Will this change be an issue for anyone using this with existing PVs that get remounted with new uid/gids?

Reply:
not if they're using the chart defaults, which were setting it to `1000` at runtime anyways